OvmfPkg/OvmfPkgX64.dsc

```diff
@@ -808,7 +816,10 @@
   #
   MdeModulePkg/Universal/Acpi/AcpiTableDxe/AcpiTableDxe.inf
   OvmfPkg/AcpiPlatformDxe/AcpiPlatformDxe.inf
-  OvmfPkg/AcpiTables/AcpiTables.inf
+  OvmfPkg/AcpiTables/AcpiTables.inf {
+    <BuildOptions>
+      *_GCC5_X64_CC_FLAGS = -fno-stack-protector
+  }
```

Comment:
Please remove this change and instead add the following to [BuildOptions]:
GCC:*_GCC5_X64_ASLCC_FLAGS = -fno-stack-protector